src/fonduer/candidates/candidates.py

```diff
@@ -38,11 +39,13 @@ def __init__(
         self_relations=False,
         nested_relations=False,
         symmetric_relations=True,
+        parallelism=None,
```

Comment:
Why not just set to 1?

Reply:
1 is equivalent to None in the code.
```py
        if parallelism is None or parallelism < 2:
            self.apply_st(xs, clear=clear, **kwargs)
```
I'm fine either way. Want it to be `1`?

Reply:
I think 1 is clear and can reduce the code.